packages/router/src/lib/operations/auction.ts

```diff
@@ -220,13 +275,11 @@ export const newAuction = async (
 
   amountReceived = amountReceivedInBigNum.sub(gasFeeInReceivingToken).toString();
 
-  const balance = await contractReader.getAssetBalance(receivingAssetId, receivingChainId);
-  logger.info("Got asset balance", requestContext, methodContext, { balance: balance.toString() });
-  if (balance.lt(amountReceived)) {
+  if (receiverLiquidity.lt(amountReceived)) {
```

Comment:
decimals not converted for liquidity

Reply:
getReceiverAmount returns the value with receiving asset decimals even if routerBalances has 18 decimals.  receiverLiquidity and amountReceived have same decimals right now.

Reply:
Correct :)